google/cloud/accesscontextmanager/quickstart/README.md

```diff
@@ -0,0 +1,166 @@
+
+3. Run the example, change the place holder(s) to appropriate values:
+
+   ```bash
+   bazel run :quickstart -- [...]
+   ```
+
+## Using with CMake
+
+> :warning: If you are using Windows or macOS there are additional instructions
+> at the end of this document.
+
+1. Install CMake. The package managers for most Linux distributions include a
+   package for CMake. Likewise, you can install CMake on Windows using a package
+   manager such as [chocolatey][choco-cmake-link], and on macOS using
+   [homebrew][homebrew-cmake-link]. You can also obtain the software directly
+   from the [cmake.org](https://cmake.org/download/).
+
+2. Install the dependencies with your favorite tools. As an example, if you use
+   [vcpkg](https://github.com/Microsoft/vcpkg.git):
+
+   ```bash
+   cd $HOME/vcpkg
+   ./vcpkg install google-cloud-cpp[core,accesscontextmanager]
+   ```
+
+   Note that, as it is often the case with C++ libraries, compiling these
+   dependencies may take several minutes.
+
+3. Configure CMake, if necessary, configure the directory where you installed
+   the dependencies:
+
+   ```bash
+   cd $HOME/gooogle-cloud-cpp/google/cloud/accesscontextmanager/quickstart
+   cmake -H. -B.build -DCMAKE_TOOLCHAIN_FILE=$HOME/vcpkg/scripts/buildsystems/vcpkg.cmake
+   cmake --build .build
+   ```
+
+4. Run the example, change the place holder to appropriate values:
+
+   ```bash
+   .build/quickstart [...]
+   ```
+
+## Platform Specific Notes
+
+### macOS
+
+gRPC [requires][grpc-roots-pem-bug] an environment variable to configure the
+trust store for SSL certificates, you can download and configure this using:
+
+```bash
+curl -Lo roots.pem https://pki.google.com/roots.pem
+export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH="$PWD/roots.pem"
+```
+
+To workaround a [bug in Bazel][bazel-grpc-macos-bug], gRPC requires this flag on
+macOS builds, you can add the option manually or include it in your `.bazelrc`
+file:
+
+```bash
+bazel build --copt=-DGRPC_BAZEL_BUILD ...
+```
+
+### Windows
+
+To correctly configure the MSVC runtime you should change the CMake minimum
+required version to 3.15 or add `-DCMAKE_POLICY_DEFAULT_CMP0091=NEW` to the
+CMake configuration step.
+
+Bazel tends to create very long file names and paths. You may need to use a
+short directory to store the build output, such as `c:\b`, and instruct Bazel
+to use it via:
+
+```shell
+bazel --output_user_root=c:\b build ...
+```
+
+gRPC [requires][grpc-roots-pem-bug] an environment variable to configure the
+trust store for SSL certificates, you can download and configure this using:
+
+```console
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command ^
+    (new-object System.Net.WebClient).Downloadfile( ^
+        'https://pki.google.com/roots.pem', 'roots.pem')
+set GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=%cd%\roots.pem
+```
+
+[bazel-install]: https://docs.bazel.build/versions/main/install.html
+[quickstart-link]: https://cloud.google.com/accesscontextmanager/docs/quickstart
```

Comment:
404. You want: https://cloud.google.com/access-context-manager/docs/quickstart

Reply:
Fixed.